src/models/Donation.js

```diff
@@ -18,7 +18,7 @@ class Donation extends Model {
   }
 
   static get TO_APPROVE() {
-    return 'ToApprove';
+    return 'To Approve';
```

Comment:
No, you should not change it. It's mapped to status in backend.

Reply:
fixed in commit below :point_down: